Pyrado/pyrado/environments/mujoco/__init__.py

```diff
@@ -26,22 +26,28 @@
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 
+import os
+
 import pyrado
 from pyrado.utils.input_output import print_cbt
 
-
```

Comment:
did black remove this line?

Reply:
I think so, I have the pre-commit hook installed.